src/components/checkboxFilter/index.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+
+import { chakra } from '@chakra-ui/react';
+
+import Stack from '../stack';
+import Checkbox from '../checkbox';
+
+type Item<Option> = {
+  value: Option;
+  label: string;
+};
+
+type Props<Option extends string | number> = {
+  /**
+   * Name of the filter - only used on the DOM.
+   */
+  name: string;
+  /**
+   * @param Item: { value: Option, label: string }
+   */
+  options: Item<Option>[];
+  /**
+   * The checkbox filter is a controlled component and the updated filter value must be passed in order to see the correct state.
+   */
+  checked: { [key in Option]: boolean };
+  /**
+   * Numeric value shown next to the checkbox label. Indicates how many search results are going to be visible after the checkbox has been applied.
```

Comment:
WDYT, to add inside options `count` key instead to have separate prop facets. Sth like { value: 'some', label: 'Some, count: 1}.

Reply:
I think this would be more effort because the current format matches the backend response: https://internal.dev.autoscout24.ch/api-docs/swagger-ui/?urls.primaryName=listing-search-service#/Listing%20Search/getFacets

Reply:
My impression as a consumer of the API was: oh I need to add 3 props with 'value' e.g petrol like
checked={{
  petrol: true,
}}
facets={{
 petrol: 20,
}}
options={[
  { label: 'Benzin', value: 'petrol' }
]}

Reply:
as dicussed, changed it to a single array